README.md

```diff
@@ -3560,4 +3560,9 @@ https://github.com/pyiron
 ### LCMP-Minimal
 LCMP Minimal : Debian - Caddy - MariaDB - PHP
 
-https://github.com/bibicadotnet/LCMP-Minimal
\ No newline at end of file
+https://github.com/bibicadotnet/LCMP-Minimal
+
+### salary-manager-solution
+Salary Manage : Laravel - PHP - MariaDB
+
```

Comment:
This is a free salary management and salary payment solution for non-profit organizations